### PR TITLE
Switch out logging package

### PR DESCRIPTION
**Pull request by aalu1418**
Swapping out Winston for Pino logging

Pino has functionality for redacting information!

First mentioned as a result of this PR: https://github.com/smartcontractkit/external-adapters-js/pull/508

Details of changes:
* Using Pino instead of Winston
* Implemented Pino redact functionality for general logging
* Using related Pino function to process URLs in WS metrics
* Using Pino dependency (fast-redact) to use Pino redact settings to redact WS redux-logger

Lays the ground work for better integration of Pino logging while also meeting the short term need of redacting API keys from WS metrics + logging. But will need a better plan to integrate general logging and WS logging

### CODE REVIEW

**Review by HenryNguyen5 — DISMISSED** [review]
🔥

File: packages/core/bootstrap/src/lib/external-adapter/logger.ts [hunk 1]
```diff
@@ -1,23 +1,66 @@
...
+}
 
-export const logger = createLogger({
+export const logger = pino({
   level: process.env.LOG_LEVEL || 'info',
-  format:
-    process.env.NODE_ENV === 'development'
-      ? combine(instanceId(), timestamp(), json(), prettyPrint())
-      : combine(instanceId(), timestamp(), json()),
-  transports: [new transports.Console()],
```
[the remark this answers is not in the document]
Comment (aalu1418) [reply packages/core/bootstrap/src/lib/external-adapter/logger.ts:22]: yeah, I think so too

File: packages/core/bootstrap/src/lib/external-adapter/logger.ts [hunk 2]
```diff
@@ -1,23 +1,65 @@
...
-const { combine, timestamp, json, prettyPrint } = format
+import { uuid } from '../util'
+import pino from 'pino'
 
-// We generate an UUID per instance and add it to the logs
-const uuid = () => {
-  if (!process.env.UUID) process.env.UUID = uuidv4()
-  return process.env.UUID
-}
+export const paths = [
```
Comment (krebernisak) [inline packages/core/bootstrap/src/lib/external-adapter/logger.ts:4]: This should be exposed by the WS module, not here.

**Review by krebernisak — COMMENTED** [review]
Nice, @aalu1418! 💯

File: packages/core/bootstrap/src/lib/external-adapter/logger.ts (hunk 2 above)
Comment (aalu1418) [reply packages/core/bootstrap/src/lib/external-adapter/logger.ts]: Moved the paths to `ws/config` and reimported them here.
Future redaction paths can have the specific paths written in the module, and imported to the logger module

Comment (krebernisak) [reply packages/core/bootstrap/src/lib/external-adapter/logger.ts]: Awesome, thx!

**Review by krebernisak — APPROVED** [review]

**Pull request merged**